.github/workflows/pull_request.workflow.yml

```
@@ -20,19 +19,18 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
-      - uses: actions/setup-node@v1.4.4
+      - uses: actions/setup-node@v4
         with:
-          node-version: "14"
+          node-version: "18.17.1"
```

Is it not enough to just use the latest minor of Node.js 18?
```suggestion
          node-version: "18"
```

---

I tried, it seems that mocha, typescript and ESM module are bugged with the last version of Node 18, but it is working in the targeted 18.17.1 idk why ...